### DISCUSSION TITLE
Trying out Rowy

### DISCUSSION THREAD

**Discussion by nonoumasy** [AntlerVC/firetable#740] [Support / Q&A] [answered]
Hello, 

First of all, thanks for creating a tool to fix a real-world issue with Firestore which is adding data in a semi-automated way.
I would like to try out Rowy for my site: 
https://history-maps.com/

My goal is to accelerate adding of content to Firestore without manually adding them on forms.
I have a few questions.
1. is there a way to geocode a column? for eg, if i have a column of locations, can that be geocode and have the latitude and longitude on two other columns? this is not a biggie and a 'nice to have'
2. My data has subcollections. is it easy to add with Rowy?

**Comment by harinij** [comment, marked as answer]
Hi @nonoumasy 👋 glad to hear you are trying Rowy. Your website looks super interesting!

To answer your questions:
1. We have a column type called `geolocation` that can be used for saving latitude and longitude information. We also have flexible column type called derivative that can be used to listen to changes in any other column and pull any info as needed. Here is demo showing the geolocation field along with a derivative column that uses an API to get the country from the geolocation. https://demo.rowy.io/table/geolocation
2. Yes, you can add subcollections to a table such that each row has a collection of data or you can also create a table for a subcollection by using the subcollection name instead of the collection name when creating a table.
